tests/puppeteer/campaigns/functional/BO/02_orders/02_invoices/03_invoiceOptions/05_enableDisableCurrentYear.js

```diff
@@ -1,6 +1,8 @@
 require('module-alias/register');
 // Using chai
 const {expect} = require('chai');
+const chai = require('chai');
+chai.use(require('chai-string'));
```

Comment:
What is that for ?

Reply:
to use function like startsWith used in this test